chapters/diffs-from-previous-editions.md

```diff
@@ -1,7 +1,20 @@
 # Annex I Differences from previous editions (Informative)
 
+# I.1 Differences between V2.3 and V2.2.2  <a name="I.1"></a>
 
-# I.1 Differences between V2.2.2 and V2.2.1 <a name="I.1"></a>
+V2.3 has added new fields to improve the ability to capture security related information and to improve interoperabiility with other SBOM formats.  
+
+Key changes include:
+* Added fields to Section 7 ( Package Information ) to describe "Primary Package Purpose" and standardize recording of "Built Date", "Release Date", "Valid Until Date".
+* Added hash algorithms (SHA3-256, SHA3-384, SHA3-512, BLAKE2b-256, BLAKE2b-384, BLAKE2b-512, BLAKE3, ADLER32 ) to the set recognized by 7.10 (Package Checksum field) and 8.4 (File checksum field)
+* Made several of the licensing properties optional rather than mandatory and requiring a “NOASSERTION” value.
+* Update Annex F ( External Repository Identifiers ) to expand security references to include advisory, fix, URL, SWID.  Expand persistent identifiers to include gitoid.
+* Update Annex G ( SPDX Lite Profile ) to include NTIA SBOM mandatory minimum fields as required.
+* Added Annex K (How To Use SPDX in Different Scenarios) to illustrate linking to external security information, and illustrate how the NTIA SBOM mandatory minimum elements map to SPDX fields.
```

Comment:
Nit - noticed an inconsistency in spaces after/before parenthesis

Reply:
FIxed.